fastly/fixtures/package/get.yaml

```diff
@@ -6,21 +6,21 @@ interactions:
     form: {}
     headers:
       User-Agent:
-      - FastlyGo/2.1.0 (+github.com/fastly/go-fastly; go1.15.4)
-    url: https://api.fastly.com/service/7i6HN3TK9wS159v2gPAZ8A/version/2/package
+      - FastlyGo/7.0.0 (+github.com/fastly/go-fastly; go1.18.3)
+    url: https://api.fastly.com/service/pChqKASI98hryShq2mlDf6/version/2/package
```

Comment:
If you could run `make fix-fixtures FASTLY_TEST_SERVICE_ID="pChqKASI98hryShq2mlDf6"` and that should replace any reference to your Service ID with the go-fastly generic one.

Reply:
I was pretty sure that I executed it 🤦. Probably I messed up between different test runs. Anyway, fixed and ready to review!